easybuild/easyconfigs/p/Perl/Perl-5.28.0-GCCcore-7.3.0.eb

```diff
@@ -18,6 +18,11 @@ dependencies = [
     ('expat', '2.2.5'),  # for XML::Parser
 ]
 
+osdependencies = [
+    ('openssl'),						# for Net::SSLeay
+    ('openssl-devel', 'libssl-dev', 'libopenssl-devel')		# dtto
+]
```

Comment:
@jose-d Since this is really one thing for one particular extension (and you have a typo in `ditto`), why not go with:

```python
# OpenSSL is required for Net::SSLeay
osdependencies = ['openssl', ('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```

I suspect `'openssl'` is not the right package name across the different Linux distros though... I guess we'll also have to consider `libssl` and `libopenssl`? cc @vanzod @akesandgren

```python
osdependencies = [('openssl', 'libssl', 'libopenssl'), ('openssl-devel', 'libssl-dev', 'libopenssl-devel')]

Reply:
@boegel 

> (typo + code style)

agree.

> I suspect `'openssl'` is not the right package name across the different Linux distros though... I guess we'll also have to consider `libssl` and `libopenssl`? cc @vanzod @akesandgren

`openssl-libs` (`libssl`|`libopenssl`) is something else, and it is actually RPM dependency of already present `openssl-devel` (and I _assume_ that other distros have same concept with their `libssl-dev`|`libopenssl-devel`) - so here we should be OK.

`Net::ssleay` is actually searching real `openssl` - not the libs.

I checked centos7 and recent ubuntu - on both it is package called `openssl`.

Reply:
OK, just a reformat should be fine then...